go/cmd/vtbackup/vtbackup.go

```diff
@@ -249,6 +249,7 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 		TopoServer:   topoServer,
 		Keyspace:     *initKeyspace,
 		Shard:        *initShard,
+		TabletAlias:  topoproto.TabletAliasString(tabletAlias),
```

Comment:
question for both BackupParams and RestoreParams -- does it make sense to bake the backup time / target snapshot time into there instead of passing as a separate arg in places where we need both the params object and the time?

Reply:
good suggestion. I'm really pleased with how compact the interface is becoming with pushing all these function parameters into params objects.